utils/wasm-builder/src/wasm_project.rs

```diff
@@ -52,6 +52,10 @@ impl WasmProject {
             .components()
             .rev()
             .take_while(|c| c.as_os_str() != "target")
+            .collect::<Vec<_>>()
+            .into_iter()
+            .rev()
+            .take_while(|c| c.as_os_str() != "build")
```

Comment:
The overall expression looks too complicated. Maybe it is worth to rework it.

Reply:
Agree. I will introduce a variable to improve readability a bit.